assignment.md

```diff
@@ -1,23 +1,31 @@
-What is the difference between push, pull, and fetch?
+# Important Git commands to understand 
 
```

Comment:
1.  Changed the title and added a # h1.   What do you think of the new heading title?